lego/apps/podcasts/views.py

```diff
@@ -0,0 +1,22 @@
+from rest_framework import viewsets
+
+from lego.apps.permissions.api.views import AllowedPermissionsMixin
+from lego.apps.podcasts.models import Podcast
+from lego.apps.podcasts.serializers import (
+    DetailedPodcastSerializer, PodcastCreateAndUpdateSerializer, PodcastSerializer
+)
+
+
+class PodcastViewSet(AllowedPermissionsMixin, viewsets.ModelViewSet):
+
+    ordering = 'created_at'
+
+    queryset = Podcast.objects.all()
+
+    def get_serializer_class(self):
+        print(self.action)
```

Comment:
Guess this isn't necessary in production? :thinking:

Reply:
Ohh, missed that one.